src/blob_storage.h

```diff
@@ -32,6 +32,7 @@ class BlobStorage {
               std::shared_ptr<BlobFileCache> _file_cache, TitanStats* stats)
       : db_options_(_db_options),
         cf_options_(_cf_options),
+        blob_run_mode_(_cf_options.blob_run_mode),
```

Comment:
Let's get cf options from somewhere, instead of setting it specifically. Maybe need to update other options in the future.

Reply:
I feel like this needs a larger refactor. MutableCFOptions only contains blob_run_mode at the moment, and table factory already uses this approach. Apart from this case, it's tedious to get a latest cf_options in Titan. A better solution would be to only care about Titan's private options without considering keeping the other parts up-to-date with RocksDB.